README.md

```diff
@@ -74,15 +74,19 @@ Proc. IEEE (2014), 102(10):1470-1484.
 <br><br>
 <a name="neuromorphic-systems"></a>
 # Neuromorphic Systems
+- Serrano-Gotarredona, T. , Andreou, A.G. , and Linares-Barranco, B., *[AER Image Filtering Architecture for Vision Processing Systems](https://doi.org/10.1109/81.788808)*, IEEE Trans. Circuits and Systems (Part I): Fundamental Theory and Applications, 46(9):1064-1071, 1999.
```

Comment:
For future additions, we put the authors, paper title and journal/conference name in three different lines.
I will take care of this